### PR TITLE
submitted code

### PR DESCRIPTION
**Pull request by xiek601**
Step 1 User Story has been completed. The following code

### CODE REVIEW

**Comment by jhonny-me** [comment]
Hi, @xiek601 can you tell us your company name here?

**Comment by jhonny-me** [comment]
Also, you can put your answer for Step2 https://github.com/Wiredcraft/test-wechat#step-2 here.

**Comment by xiek601** [comment]
Of course, my company is called ycode

**Comment by jhonny-me** [comment]
@xiek601 please don't forget the test step2.

File: pages/search/search.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      console.log(event.detail.value);
+      console.log(that);
+      that.searchRequest(event.detail.value);
+    }, 300)();
+  },
+  //搜索请求
+  searchRequest: function (value) {
+    let temp = this;
+    wx.request({
+      url: 'https://api.github.com/search/repositories',
```
Comment (shuaibird) [inline pages/search/search.js:31]: I think you may use the wrong API here. The requirement is to list organization repos, so the API to use should be this one: https://developer.github.com/v3/repos/#list-organization-repositories

File: pages/search/search.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import Tool from '../../utils/tool.js';
```
Comment (shuaibird) [inline pages/search/search.js:1]: Why capitalize the name of this object?

File: pages/search/search.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    //查询函数防抖动
+    Tool.debounce(function () {
+      console.log(event.detail.value);
+      console.log(that);
+      that.searchRequest(event.detail.value);
+    }, 300)();
+  },
+  //搜索请求
+  searchRequest: function (value) {
+    let temp = this;
```
Comment (shuaibird) [inline pages/search/search.js:29]: IMHO, you should keep a consistent naming convention for the whole project.

File: utils/util.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+const formatTime = date => {
```
Comment (shuaibird) [inline utils/util.js:1]: No testing for the util function(s)?

File: pages/searchDetail/searchDetail.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  },
+
+//  获取数据方法
+  getData(param){
+    let that = this;
+    wx.showLoading({
+      title: '加载中',
+    })
+    wx.request({
+      url: 'https://api.github.com/repos/' + param,
```
Comment (shuaibird) [inline pages/searchDetail/searchDetail.js:26]: Would it break if the request responding with `404`?
There's a quirk in the `wx.request` API, check the DOC.

**Pull request closed without merging**